### PR TITLE
[TwigComponent] Optimize the ComponentLoggerListener

### PR DESCRIPTION
**Pull request by smnandre**
* Clean unused code
* Reduce memory usage
* Optimize class/templates lookup

### CODE REVIEW

File: src/TwigComponent/src/EventListener/TwigComponentLoggerListener.php [hunk 1]
```diff
@@ -12,90 +12,55 @@
...
-use Symfony\UX\TwigComponent\Event\PreCreateForRenderEvent;
-use Symfony\UX\TwigComponent\Event\PreMountEvent;
 use Symfony\UX\TwigComponent\Event\PreRenderEvent;
 
 /**
  * @author Simon André <smn.andre@gmail.com>
+ *
+ * @internal
  */
 class TwigComponentLoggerListener implements EventSubscriberInterface, ResetInterface
```
Comment (kbond) [inline src/TwigComponent/src/EventListener/TwigComponentLoggerListener.php]:
```suggestion
final class TwigComponentLoggerListener implements EventSubscriberInterface, ResetInterface
```

**Review by kbond — APPROVED** [review]
This is great!

**Comment by javiereguiluz** [comment]
Nice! Maybe we can also remove the `symfony/stopwatch` dev dependency here?

https://github.com/symfony/ux/blob/8dd706da2c1ada2174c920c246daa6765b176cef/src/TwigComponent/composer.json#L43

**Review by Kocal — APPROVED** [review]

**Comment by smnandre** [comment]
> Nice! Maybe we can also remove the `symfony/stopwatch` dev dependency here?
> 
> https://github.com/symfony/ux/blob/8dd706da2c1ada2174c920c246daa6765b176cef/src/TwigComponent/composer.json#L43

Absolutely, thank you for catching this!

**Comment by Kocal** [comment]
Do... you think we can have some Blackfire profiles to show the performance improvments? 🙏🏻

**Comment by smnandre** [comment]
> Do... you think we can have some Blackfire profiles to show the performance improvments? 🙏🏻

I did not have time yet to configure it 😶‍🌫️  

But i will in the next few days (already started a dedicated repo to measure performance of twig/live components)

**Pull request merged**